src/widgets/trOverview.ts

```diff
@@ -87,7 +87,12 @@ export default async function trOverview(
     where: {
       [Op.and]: [
         {
-          'data.status': TRAINING_REPORT_STATUSES.COMPLETE,
```

Comment:
Was this something they wanted changed? (just a double check)

Reply:
Yes, they wrote a ticket specifically for it, TTAHUB-2786